failed to build using mvn

I'm fresh to java&mvn, after checkout the repo and  kickoff mvn package, an error occurs.

    [ERROR] Failed to execute goal on project hdfs-cli: Could not resolve dependencies for project com.instanceone.hdfs:hdfs-cli:jar:0.0.1-SNAPSHOT: Could not find artifact com.github.ptgoetz:stemshell:jar:0.1.0-SNAPSHOT -> [Help 1]

Would you like to give some instructions about building and using this tools?

Thank you very much!

--- a/pom.xml
+++ b/pom.xml
@@ -13,7 +13,7 @@
 		<dependency>
 			<groupId>com.github.ptgoetz</groupId>
 			<artifactId>stemshell</artifactId>
-			<version>0.1.0-SNAPSHOT</version>
+			<version>0.1.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.hadoop</groupId>

